Doc/tools/touch-clean-files.py

```diff
@@ -0,0 +1,21 @@
+#!/usr/bin/env python3
+"""
+Touch files that must pass Sphinx nit-picky mode
+so they are rebuilt and we can catch regressions.
+"""
+
+from pathlib import Path
+
+# Input file has blank line between entries to reduce merge conflicts
+with Path("Doc/tools/clean-files.txt").open() as clean_files:
+    CLEAN = [
+        filename.strip()
+        for filename in clean_files
+        if filename.strip() and not filename.startswith("#")
+    ]
+
+print("Touching:")
+for filename in CLEAN:
+    if filename:
+        print(filename)
+        Path(filename).touch()
```

Comment:
```suggestion
    print(filename)
    filename.touch()
```

* You're already checking `if filename` above, so it can be removed here
* Also, I moved the `Path()` coercion above, with the rest